SmartDeviceLink/public/SDLShow.h

```diff
@@ -231,7 +251,7 @@ NS_ASSUME_NONNULL_BEGIN
  *            will not display anything)</li>
  *            </ul>
  */
-@property (strong, nonatomic, nullable) NSString *mediaClock;
+@property (strong, nonatomic, nullable) NSString *mediaClock __deprecated;
```

Comment:
The generated comment for this should say something about when it was deprecated. Please include that.